javagen/src/main/java/com/azure/autorest/template/TemplateHelper.java

```diff
@@ -49,9 +49,30 @@ public static void createHttpPipelineMethod(JavaSettings settings, String defaul
     }
 
     private static void createGenericHttpPipelineMethod(JavaSettings settings, String defaultCredentialScopes, SecurityInfo securityInfo, PipelinePolicyDetails pipelinePolicyDetails, JavaBlock function) {
-        function.line("HttpPipeline httpPipeline = HttpPipelineBuilder.createDefaultPipeline();");
-        function.methodReturn("httpPipeline");
-        // TODO: default pipeline is immutable and we need to add KeyCredential policy to the pipeline depending on securityInfo.
+        // TODO: generic
```

Comment:
I am using `TODO: generic` for things that need further work when generic-core be more stable.